exporter/sumologicexporter/README.md

```diff
@@ -0,0 +1,50 @@
+# Sumo Logic Exporter
+
+This exporter supports sending logs and metrics data to [Sumo Logic](https://www.sumologic.com/).
+
+The following configuration options are supported:
+
+- `url` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
+- `compress` (optional): Option to enable compression (default true)
+- `compress_encoding` (optional): Compression encoding format, either gzip or deflate (default gzip)
+- `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default 1MB is recommended.
+- `metadata_attributes` (optional): List of regexes for attributes which should be send as metadata
+- `log_format` (optional) (logs only): Format to post logs into Sumo. (default `json`)
+  - `text` - Logs will appear in Sumo Logic in text format
+  - `json` - Logs will appear in Sumo Logic in json format.
```

Comment:
```suggestion
- `log_format` (optional) (logs only): Format to use when sending logs to Sumo. (default `json`) (possible values: `json`, `text`)
```

I have a suggestion of changing the structure of those bullet points into a list of values in parenthesis i.e. "possible values: `text` or `json`".

Reason for this is that other configuration options that have nested bullet points have a structure e.g. `retry_on_failure` and `sending_queue` have fields that can be customized while `log_format` is a key that requires a string value to be set (not a map/dict)